cloudify_nsx/library/nsx_security_tag.py

```diff
@@ -51,7 +51,20 @@ def delete_tag(client_session, resource_id):
     common.check_raw_result(result)
 
 
+def tag_vm_to_resource_id(tag_id, vm_id):
+    """Generate resource_id from tag_id/vm_id"""
+    if not vm_id or not tag_id:
+        raise cfy_exc.RecoverableError(
+            "Please recheck tag_id/vm_id"
```

Comment:
I think about possibility to have this ID on next attempt  with getting id from get_attributes in cloudify.nsx.security_tag_vm node, but it is really unreal situation.